src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/DruidCombatAI.java

```diff
@@ -0,0 +1,347 @@
+        combatPathingOptions.setEnterDoors(true);
+        combatPathingOptions.setCanOpenDoors(true);
+        combatPathingOptions.setCanSwim(true);
+        combatPathingOptions.withOnPathCost(0.8);
+        combatPathingOptions.withJumpCost(0.01);
+        combatPathingOptions.withDropCost(0.9);
+    }
+
+    @Override
+    protected void doAttack(final LivingEntity target)
+    {
+        if (user.distanceToSqr(target) < RANGED_FLEE_SQDIST)
+        {
+            if (user.getRandom().nextInt(FLEE_CHANCE) == 0 &&
+                  !((AbstractBuildingGuards) user.getCitizenData().getWorkBuilding()).getTask().equals(GuardTaskSetting.GUARD))
+            {
+                user.getNavigation().moveAwayFromLivingEntity(target, getAttackDistance() / 2.0, getCombatMovementSpeed());
+            }
+        }
+        else
+        {
+            user.getNavigation().stop();
+        }
+
+        user.swing(Hand.MAIN_HAND);
+
+        final int level = user.getCitizenData().getCitizenSkillHandler().getLevel(ModGuardTypes.druid.getSecondarySkill());
+        final int time = user.getCitizenData().getCitizenSkillHandler().getLevel(ModGuardTypes.druid.getPrimarySkill()) * 20;
+
+        final float inaccuracy = 99f / level;
+        final Effect effect;
+        final ItemStack stack = new ItemStack(Items.SPLASH_POTION);
+        boolean gotMaterial = false;
+        BiPredicate<LivingEntity,Effect> predicate;
+        if (user.getCitizenColonyHandler().getColony().getResearchManager().getResearchEffects().getEffectStrength(DRUID_USE_POTIONS) > 0
+              && InventoryUtils.hasItemInItemHandler(user.getInventoryCitizen(), item -> item.getItem() instanceof PotionItem)
+              && InventoryUtils.hasItemInItemHandler(user.getInventoryCitizen(), item -> item.getItem() == ModItems.mistletoe))
+        {
+            gotMaterial = true;
+        }
+        if (AbstractEntityAIGuard.isAttackableTarget(user, target))
+        {
+            effect = ADVERSE_EFFECTS.get(user.getRandom().nextInt(gotMaterial ? 2 : 1));
+            predicate = (entity, eff) -> AbstractEntityAIGuard.isAttackableTarget(user, entity);
+        }
+        else
+        {
+            effect = SUPPORT_EFFECTS.get(user.getRandom().nextInt(gotMaterial ? 4 : 1));
+            predicate = (entity, eff) -> !AbstractEntityAIGuard.isAttackableTarget(user, entity);
+        }
+
+        PotionUtils.setCustomEffects(stack, Collections.singleton(new EffectInstance(effect, time, gotMaterial ? 2 : 0)));
+        DruidPotionEntity.throwPotionAt(stack, target, user, user.getCommandSenderWorld(), POTION_VELOCITY, inaccuracy, predicate);
+
+        if (gotMaterial)
+        {
+            InventoryUtils.removeStackFromItemHandler(user.getCitizenData().getInventory(), new ItemStack(Items.POTION), 1);
+            InventoryUtils.removeStackFromItemHandler(user.getCitizenData().getInventory(), new ItemStack(ModItems.mistletoe), 1);
+        }
```

Comment:
So it always throws and consumes if items present?

Reply:
why should people give them items then? if they throw a potion anyway?

Reply:
they throw better potions

Reply:
ah got it (but how to obtain mistletoe

Reply:
Forester generates it